crates/db_views/src/post_view.rs

```diff
@@ -149,6 +163,7 @@ impl PostView {
       subscribed: follower.is_some(),
       saved: saved.is_some(),
       read: read.is_some(),
+      creator_blocked: creator_blocked.is_some(),
```

Comment:
The quirk with posts / comments is that only the list version filters out the blocked persons / communities. The individual get version will still show it, but set this blocked flag.